README.md

```diff
@@ -91,6 +94,23 @@ It can:
 - Revoke the pending guardian (which means it can revoke any attempt to change the guardian).
 - Revoke the pending cap of any market.
 
+### Idle Supply
+
+In some cases, the vault's curator or allocators may want to keep some funds "idle", to guarantee lenders that some liquidity can be withdrawn from the vault (beyond the liquidity of each of the vault's markets).
+
+To achieve this, it is advised to allocate "idle" funds to any market on Morpho Blue having:
+
+- The vault's asset as loan token.
+- No collateral token (`address(0)`).
+- An arbitrary IRM.
+- No oracle (`address(0)`).
```

Comment:
```suggestion
- An arbitrary oracle.
```

Reply:
Can I still suggest address(0)? I think it's interesting to have at least the maximum nb of parameters standard for idle markets

Reply:
https://morpholabs.slack.com/archives/C02N7CZ088N/p1700002028695799?thread_ts=1699975938.520309&channel=C02N7CZ088N&message_ts=1700002028.695799